hello_world_on_chain/test/user.ts

```diff
@@ -1,79 +1,92 @@
+// noinspection DuplicatedCode
```

Comment:
Goland keeps giving me warnings about "duplicated code" highlighting that portions of this test file is the same in our other example folders. it's super annoying so I added this to turn it off